tests/example_app/example/page1/templates/page_1.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+  <br/>
+  <a href="/page-1/a">page 1 A</a>
+  <a href="/page-1/b">page 1 B</a>
+
+  <h2>Currently on page 1</h2>
+
+  <!-- relative static ".static" -->
```

Comment:
You can remove all of these HTML comments in this file and `page_2.html`.

Reply:
[d06ab59 favoring double quotes and removing html comments](https://github.com/nickjj/flask-static-digest/pull/33/commits/d06ab598dfca84d46029872fa82baf6f77884cfa)